tests/test_cases/test_discovery/test_discovery.py

```diff
@@ -199,35 +219,35 @@ def access_string(dut):
     test_string = "cocotb"
     dut.stream_in_string.setimmediatevalue(test_string)
 
-    variable_string = dut.stream_out_string
```

Comment:
Was this change intentional? "Variable" is the correct spelling.

Reply:
Oh, thanks, I somehow reverted a change of my own from another PR. I'll rework this next week.